src/Loader/LoaderInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Tuf\Loader;
+
+use Psr\Http\Message\StreamInterface;
+
+/**
+ * Defines an interface to load data as a stream.
+ *
+ * The data source can be anything, from anywhere, but it must be returned as an
+ * implementation of \Psr\Http\Message\StreamInterface.
+ *
+ * This is an internal interface used to load untrusted data! External code
+ * should not use this directly, but instead rely on \Tuf\Client\Updater to
+ * load TUF-verified data.
+ */
+interface LoaderInterface
+{
+    /**
+     * Loads data as a stream.
+     *
+     * @param string $uri
+     *   The URI of the data to load. The meaning of this depends on the
+     *   implementing class; it could be a URL, a relative or absolute file
+     *   path, or something else.
```

Comment:
I don't think a `URI` is right name for this.  URI is does a definition and I don't think a "relative file path" fall into this this. could we change this to "locator" especially since we include "something else" as a possible value

https://stackoverflow.com/questions/44692872/is-a-local-file-path-a-uri